test/screenshotter/ss_data.yaml

```diff
@@ -106,6 +106,8 @@ SqrtRoot: |
 SupSubCharacterBox: a_2f_2{f}_2{aa}_2{af}_2
 SupSubHorizSpacing: |
     x^{x^{x}}\Big|x_{x_{x_{x_{x}}}}\bigg|x^{x^{x_{x_{x_{x_{x}}}}}}\bigg|
+SupSubLeftAlignReset: |
```

Comment:
Could you add a plain `\omega^{8}_{888}` not in a fraction here too, since those environments are apparently different?